packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -1381,50 +1399,116 @@ function collectRecommendations({
   };
 }
 
-// Finds functions declared as dependencies
+// If the node will result in constructing a referentially unique value, return
+// its human readable type name, else return null.
+function getConstructionExpresionType(node) {
```

Comment:
Nit: `Expression`

Reply:
Fixed in https://github.com/facebook/react/pull/19590/commits/eaa2aff9424e7ded164799fa2aef27ccfa4ed3d0